lib/Runtime/Types/CMakeLists.txt

```diff
@@ -11,17 +8,17 @@ add_library (Chakra.Runtime.Types
     DynamicObjectSnapshotEnumerator.cpp
     DynamicObjectSnapshotEnumeratorWPCache.cpp
```

Comment:
This file still has build errors for me- please comment it out and fix it in the next commit